### PR TITLE
Clean travis related

### PR DESCRIPTION
**Pull request by fonsecadeline**
I am not sure what we should do in _docker-compose.yml_ file.

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '3.3'
 services:
   redis:
     image: redis:${REDIS_VERSION:-3.2-alpine}
-    hostname: travis-redis
+    hostname: travis-redis # should we rename ? remove ?
```
Comment (braktar) [inline docker/docker-compose.yml:5]: Did you try to remove the hostname ? If so, does it have an impact on the CI ?

Comment (alain-andre) [reply docker/docker-compose.yml:5]: There is no impact. This was only to specify this compose file was for CI tests only.

File: test/lib/heuristics/dichotomious_test.rb [hunk 2]
```diff
@@ -23,11 +23,6 @@ def test_dichotomious_approach
       vrp = TestHelper.load_vrp(self)
       vrp.resolution_dicho_algorithm_service_limit = 457 # There are 458 services in the instance. TODO: Remove it once the dicho contions are stabilized
 
-      if ENV['TRAVIS'] # Compansate for travis performance
```
Comment (alain-andre) [inline test/lib/heuristics/dichotomious_test.rb:26]: Do we need specifying durations on CI Github Action ?  We can use `ENV['CI']` in place of `ENV['TRAVIS']`

File: test/rubocop_test.rb [hunk 3]
```diff
@@ -19,7 +19,7 @@
 
 class RubocopTest < Minitest::Test
   def test_no_error_in_files
-    parallel = ENV['RUBOCOP_PARALLEL'] || ENV['TRAVIS'] ? '--parallel' : nil
```
Comment (alain-andre) [inline test/rubocop_test.rb:22]: I think running rubocop in parallel on `ENV['CI']` should be a good idea.

File: test/rubocop_test.rb [hunk 4]
```diff
@@ -28,7 +28,7 @@ def test_no_error_in_files
   end
 
   def test_no_warning_in_tests
-    parallel = ENV['RUBOCOP_PARALLEL'] || ENV['TRAVIS'] ? '--parallel' : nil
```
Comment (alain-andre) [inline test/rubocop_test.rb:31]: I think running rubocop in parallel on `ENV['CI']` should be a good idea.

File: test/real_cases_dichotomious_test.rb [hunk 5]
```diff
@@ -18,7 +18,7 @@
 require './test/test_helper'
 
 class RealCasesTest < Minitest::Test
-  if !ENV['SKIP_REAL_DICHO'] && !ENV['SKIP_DICHO'] && !ENV['TRAVIS']
```
Comment (alain-andre) [inline test/real_cases_dichotomious_test.rb:21]: If we need to skip this test on CI we can replace with `ENV['CI']`

File: test/lib/heuristics/dichotomious_test.rb (hunk 2 above)
Comment (fonsecadeline) [reply test/lib/heuristics/dichotomious_test.rb:26]: Tests work as is right now so I don't think we need anything like that for now.

File: docker/docker-compose.yml (hunk 1 above)
Comment (alain-andre) [reply docker/docker-compose.yml:5]: Yes we can remove hostnames on each service. I would better change `travis` by `ci`

File: .github/workflows/main.yml [hunk 6]
```diff
@@ -16,7 +16,7 @@ jobs:
   test_basis:
     runs-on: ubuntu-latest
     env:
-      OPTIONS: "COV=false LOG_LEVEL=info SKIP_DICHO=true SKIP_REAL_CASES=true SKIP_PERIODIC=true SKIP_SPLIT_CLUSTERING=true"
+      OPTIONS: "CI=true COV=false LOG_LEVEL=info SKIP_DICHO=true SKIP_REAL_CASES=true SKIP_PERIODIC=true SKIP_SPLIT_CLUSTERING=true"
```
Comment (alain-andre) [inline .github/workflows/main.yml:19]: This is not necessary, the environment variable *CI* is always true in Github Action : https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables Sorry if I wasn't clear. You can remove for all jobs.

File: test/real_cases_dichotomious_test.rb (hunk 5 above)
Comment (alain-andre) [reply test/real_cases_dichotomious_test.rb:21]: *RealCasesTest* must be launched in pipeline ? If not we must add `&& !ENV['CI']`

**Review by alain-andre — CHANGES_REQUESTED** [review]

**Review by braktar — APPROVED** [review]

**Review by senhalil — APPROVED** [review]
Intensive option increases the test duration, timeout limits need to be corrected accordingly

**Review by alain-andre — APPROVED** [review]

**Comment by senhalil** [comment]
@fonsecadeline it looks like this needs manual rebase since workflow/main.yml is modified. Can you do the rebase and force push? We can merge this if the time-out limits are alright :crossed_fingers:

**Pull request merged**